doc/docs/reference/upgrading.md

```diff
@@ -3,6 +3,17 @@
 This document provides instructions for specific steps required to upgrading your Opal
 application to a later version where there are extra steps required.
 
+### 0.10.1 -> 0.10.2
+
+The previous Django upgrade stopped compressor minifying files When DEBUG is set to False.
```

Comment:
I don't think you need to talk about the problem here - this document is 'what do I need to do' e.g. install an updated dependency. 

This text doesn't actually tell me that I need to upgrade something, or that I'll have to update my requirements.txt though - the audience for upgrading.md is someone not following Opal development closely, and potentially requiring detailed instructions to work through the steps required...